src/probes/crash_probe.c

```diff
@@ -443,12 +444,18 @@ static bool is_banned_path(char *fullpath)
         }
 
         // Anything outside of /usr/, or in /usr/local/, we consider third-party
-        if (g_regex_match_simple("^[!/]usr[!/]", fullpath, 0, 0) == FALSE) {
-                return true;
-        } else if (g_regex_match_simple("^[!/]usr[!/]local[!/]", fullpath,
-                                        0, 0) == TRUE) {
-                return true;
-        }
+
+	if (strncmp(fullpath, "/usr/", 5) != 0)
+		return true;
+
+	if (strncmp(fullpath, "!usr!", 5) != 0)
+		return true;
+
+	if (strncmp(fullpath,"/usr/local/", 11) == 0)
+		return true;
+	if (strncmp(fullpath,"!usr!local!", 11) == 0)
+		return true;
```

Comment:
@devimc Yes, that is one of our code style guidelines. However, I'm not opposed to simply running uncrustify to fix these types of issues later on (see scripts/source-indent.sh).